common/reviews/api/rush-lib.api.md

```diff
@@ -1229,6 +1229,10 @@ export class RushConfiguration {
     //
     // @internal (undocumented)
     readonly _rushPluginsConfiguration: RushPluginsConfiguration;
+    // Warning: (ae-forgotten-export) The symbol "IRushProjectsJson" needs to be exported by the entry point index.d.ts
+    //
+    // @internal
+    readonly rushProjectsJson: IRushProjectsJson | undefined;
```

Comment:
Rename this property to have a leading underscore.

Reply:
Changed